code/src/components/ShelfCard.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components'
+import { nutrition } from '../reducers/nutrition'
+import generalProducts from '../assets/generalProducts.png'
```

Comment:
`generalProducts` won't give the reader the hint that this is an imageUrl if read in-line.  Naming can help with this.